### PR TITLE
bash: update Spanish translation

### PR DESCRIPTION
**Pull request by tricantivu**
- [X] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [X] The page(s) have at most 8 examples.
- [X] The page description(s) have links to documentation or a homepage.
- [X] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):** `5.2.21(1)-release`

I believe there is still room for improvement for the page, for example:

- Joining the second line of the command description (i.e. `Bourne-Again SHell, un intérprete de línea de comandos compatible con sh`).

- Use defined articles (e.g. *el*) in descriptions of examples.

- Indicate that not all failed commands will cause `bash` to exit when invoked with the option `-e`, for example:

```shell
nonexisting-command | nonexisting-command | sleep 3600  
```

- Translate the placeholders completely (e.g. *ruta/al/script*). Apparently, the term *script* is not defined by all dialects of Spanish ([it is defined in Mexico](https://dem.colmex.mx/Ver/script)). Thus, I am not certain how to achieve this (maybe *guión*?).

### CODE REVIEW

File: pages.es/common/bash.md [hunk 1]
```diff
@@ -2,12 +2,17 @@
 
 > Bourne-Again SHell.
 > Intérprete de línea de comandos compatible con `sh`.
+> Vea también: `zsh`, `histexpand` (expansión de historial de comandos).
```
Comment (vitorhcl) [inline pages.es/common/bash.md:5]:
```suggestion
> Vea también: `zsh`; `histexpand`, para expansión de historial de comandos.
```
This is the right formatting according to the section added by #11868.

**Review by vitorhcl — CHANGES_REQUESTED** [review]

**Comment by vitorhcl** [comment]
@tricantivu I was seeing the file and I remembered the verb tense inconsistency issue in the Portuguese translations. Many files there had some verbs in the infinitive, so maybe you could add the rule that verbs should be in third person singular present indicative tense in Spanish specific rules as well and fix this inconsistency.

**Comment by vitorhcl** [comment]
> - Joining the second line of the command description (i.e. `Bourne-Again SHell, un intérprete de línea de comandos compatible con sh`).

Agreed, the English page is exactly this way.

**Review by vitorhcl — APPROVED** [review]

**Review by kant — APPROVED** [review]
LGTM

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Comment by tricantivu** [comment]
> @tricantivu I was seeing the file and I remembered the verb tense inconsistency issue in the Portuguese translations. Many files there had some verbs in the infinitive, so maybe you could add the rule that verbs should be in third person singular present indicative tense in Spanish specific rules as well and fix this inconsistency.

Could you please show me an example of the inconsistency?

I am not fluent in Portuguese.

**Comment by vitorhcl** [comment]
> > @tricantivu I was seeing the file and I remembered the verb tense inconsistency issue in the Portuguese translations. Many files there had some verbs in the infinitive, so maybe you could add the rule that verbs should be in third person singular present indicative tense in Spanish specific rules as well and fix this inconsistency.
> 
> Could you please show me an example of the inconsistency?
> 
> I am not fluent in Portuguese.

@tricantivu Sure! Spanish is a quite similar in some aspects. For example, there is a `ls` example:

"Lista arquivos um por linha:"

If this was in the infinitive, it would be "Listar arquivos um por linha:"

There's a example of inconsistency on pages.es/osx/gtee:

"- Ver documentación para el comando original:"

**Comment by kant** [comment]
So: “Ver documentación para el comando original:” > “Vea la documentación para el comando original:”

**Comment by vitorhcl** [comment]
> So: “Ver documentación para el comando original:” > “Vea la documentación para el comando original:”

@tricantivu exactly :) and it would be good to document this, maybe even do a single section for all languages that use this verbal tense

**Comment by tricantivu** [comment]
> > So: “Ver documentación para el comando original:” > “Vea la documentación para el comando original:”
> 
> @tricantivu exactly :) and it would be good to document this, maybe even do a single section for all languages that use this verbal tense

Alright, I will start writing.

**Pull request merged**